packages/styled-components/src/native/index.js

```diff
@@ -1,7 +1,207 @@
 // @flow
 
 /* eslint-disable import/no-unresolved */
-import * as reactNative from 'react-native';
+import {
```

Comment:
Only a few of these modules are components that can be styled. Perhaps a limited subset could be imported instead? It might also be worth reassessing the "alias" API altogether. Leaving it up to consumers to do `styled(View)` instead seems like it would play better with lazy eval and load, as well as avoiding issues related to future component deprecations in React Native needing to be fixed in this library as well as any app code depending on it.

Reply:
I limited it to the ones which are aliased now

Reply:
Using `import * as reactNative from react-native` will always include every react native web module. This is something we really want to avoid.